translations/harbour-seaprint-de.ts

```diff
@@ -613,1234 +613,1201 @@ auf diesem Drucker</translation>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Nr. 14</translation>
     </message>
     <message>
         <source>Envelope #9</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Nr. 9</translation>
     </message>
     <message>
         <source>8.5 x 13.4″</source>
-        <translation type="unfinished"></translation>
+        <translation>8,5 x 13,4 Zoll</translation>
     </message>
     <message>
         <source>Envelope Personal</source>
-        <translation type="unfinished"></translation>
+        <translation>Umschlag Personal</translation>
     </message>
     <message>
         <source>Quarto</source>
-        <translation type="unfinished"></translation>
+        <translation>Quarto</translation>
     </message>
     <message>
         <source>8.94 x 14″</source>
-        <translation type="unfinished"></translation>
+        <translation>8,94 x 14 Zoll</translation>
     </message>
     <message>
         <source>13 x 19″</source>
-        <translation type="unfinished"></translation>
+        <translation>13 x 19 Zoll</translation>
     </message>
     <message>
         <source>30 x 42″</source>
-        <translation type="unfinished"></translation>
+        <translation>30 x 42 Zoll</translation>
     </message>
     <message>
         <source>12 x 16″</source>
-        <translation type="unfinished"></translation>
+        <translation>12 x 16 Zoll</translation>
     </message>
     <message>
         <source>14 x 17″</source>
-        <translation type="unfinished"></translation>
+        <translation>14 x 17 Zoll</translation>
     </message>
     <message>
         <source>18 x 22″</source>
-        <translation type="unfinished"></translation>
+        <translation>18 x 22 Zoll</translation>
     </message>
     <message>
         <source>17 x 24″</source>
-        <translation type="unfinished"></translation>
+        <translation>17 x 24 Zoll</translation>
     </message>
     <message>
         <source>2 x 3.5″</source>
-        <translation type="unfinished"></translation>
+        <translation>2 x 3,5 Zoll</translation>
     </message>
     <message>
         <source>10 x 12″</source>
-        <translation type="unfinished"></translation>
+        <translation>10 x 12 Zoll</translation>
     </message>
     <message>
         <source>20 x 24″</source>
-        <translation type="unfinished"></translation>
+        <translation>20 x 24 Zoll</translation>
     </message>
     <message>
         <source>3.5 x 5″</source>
-        <translation type="unfinished"></translation>
+        <translation>3,5 x 5 Zoll</translation>
     </message>
     <message>
         <source>4 x 4″</source>
-        <translation type="unfinished"></translation>
+        <translation>4 x 4 Zoll</translation>
     </message>
     <message>
         <source>5 x 5″</source>
-        <translation type="unfinished"></translation>
```

Comment:
What happened here? Why are there missing rows?

Reply:
I don´t know. I just opened the file and the rows exists.